.gitignore

```diff
@@ -22,3 +22,6 @@
 
 releases/
 *__pycache__/
+
+#VSCode
+.vscode
```

Comment:
We want to keep the .vscode folder in the repo because it contains the settings for the [Lua plugin](https://marketplace.visualstudio.com/items?itemName=sumneko.lua).

```suggestion
```